Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -356,3 +356,7 @@ private func fetchProject(projectOrParam: Either<Project, Param>, shouldPrefix:
 
   return projectProducer
 }
+
+private func shouldGoToManagePledge(with type: PledgeStateCTAType) -> Bool {
+  return type == .viewBacking || type == .manage || type == .fix
```

Comment:
Instead of all the ORs I usually prefer writing this sort of thing as:

```swift
return [.viewBacking, .manage, .fix].contains(type)
```
But I know that some folks don't like to read this because it reverses things a bit and can be confusing. But I recall seeing [this tip](https://www.swiftbysundell.com/tips/expressively-comparing-a-value-with-a-list-of-candidates/) before.

So perhaps we should add something similar:

```swift
private func shouldGoToManagePledge(with type: PledgeStateCTAType) -> Bool {
  return type.isAny(of: .viewBacking, .manage, .fix)
}

extension Equatable {
  func isAny(of candidates: Self...) -> Bool {
    return candidates.contains(self)
  }
}
```
What do you think?

Reply:
Ohh I really like that! It becomes much easier to read.